File: src/main/java/com/will/framework/OpenSessionInView.java
package com.will.framework;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by 25019 on 2015/11/7.
 */
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.METHOD})
public @interface OpenSessionInView {
}

File: src/main/java/com/will/framework/OpenSessionInView.kt
package com.will.framework

import java.lang.annotation.ElementType
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy
import java.lang.annotation.Target

/**
 * Created by zoumy on 2017/5/12 16:30.
 *
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
annotation class OpenSessionInView